velox/dwio/text/writer/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+velox_add_library(velox_dwio_text_writer TextWriter.cpp BufferedWriterSink.cpp)
+
+velox_link_libraries(
+  velox_dwio_text_writer
+  velox_dwio_arrow_parquet_writer_lib
+  velox_dwio_arrow_parquet_writer_util_lib
```

Comment:
Curious, why does text writer need to depend on arrow parquet writer lib ?
This creates problems in presto_server build because parquet arrow are not built natively on the mac. 
(Another miss on velox part is that parquet does not build on the mac, and thus velox_dwio_arrow_parquet_writer_lib is not built , along with obviously velox_dwio_text_writer) .